ci-operator/step-registry/openshift-qe/windows-bootstorm/openshift-qe-windows-bootstorm-commands.sh

```
@@ -0,0 +1,61 @@
+fi
+
+WINDOWS_IMAGE_URL=${WINDOWS_IMAGE_URL:-$(cat "${WINDOWS_SECRETS_PATH}/windows_url" 2>/dev/null || echo "")}
+if [[ -z "${WINDOWS_IMAGE_URL}" ]]; then
+    echo "ERROR: WINDOWS_IMAGE_URL is not set and windows_url not found in ${WINDOWS_SECRETS_PATH}"
+    exit 1
+fi
+
+REPO_URL="https://github.com/cloud-bulldozer/e2e-benchmarking"
+LATEST_TAG=$(git ls-remote --tags https://github.com/cloud-bulldozer/e2e-benchmarking.git | awk -F'refs/tags/' '{print $2}' | grep -v '\^{}' | sort -V | tail -n1)
+TAG_OPTION="--branch $(if [ "$E2E_VERSION" == "default" ]; then echo "$LATEST_TAG"; else echo "$E2E_VERSION"; fi)"
+git clone $REPO_URL $TAG_OPTION --depth 1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)